src/Facebook/InstantArticles/Elements/Author.php

```diff
@@ -198,7 +198,9 @@ public function toDOMElement($document = null)
         $element->appendChild($ahref);
 
         // Appends author description
-        $element->appendChild($document->createTextNode($this->description));
+        if ($this->description) {
```

Comment:
Probably a better strategy will be to use the Type::isTextEmpty, instead of just "if" also.